src/components/Todo.vue

```diff
@@ -0,0 +1,150 @@
+<template>
+  <div class="todo">
+    <template class="todo-intarface">
```

Comment:
Typo here.

`intarface` -> `interface`

Also, why do we need a `<template>` here? I don't think it does anything.